### PR TITLE
Fixing backend linting errors

### PR DESCRIPTION
**Pull request by kayladavis**
Running `yarn run format` on the frontend doesn't produce any changes

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
     "sourceMap": false,
     "instrument": false
   },
+  "eslintConfig": {
+    "env": {
+      "browser": true,
```
Comment (robmcl4) [inline backend/package.json]: erm.... I have no idea how ESlint works but this looks odd to me

File: backend/package.json [hunk 2]
```diff
@@ -74,4 +83,4 @@
     "standard": "^10.0.3",
     "supertest": "^3.0.0"
   }
-}
+}
```
Comment (robmcl4) [inline backend/package.json]: nit: every other file ends in a newline (UNIX style) ... we should also end this file in a newline

File: backend/package.json (hunk 1 above)
Comment (billdybas) [reply backend/package.json]: [Specifying Environments](https://eslint.org/docs/user-guide/configuring#specifying-environments)

"An environment defines global variables that are predefined."

`"broswer": true,` should be removed, as browser globals are not available in Node

the other config options are fine, though.

**Comment by billdybas** [comment]
@kayladavis Try `npm run format` on the frontend. I get at least 40 files changed.

File: backend/package.json (hunk 1 above)
Comment (kayladavis) [reply backend/package.json]: oops, meant to remove that. I was pulling in stuff from the config page and forgot.

**Pull request merged**